src/extension/background-script/actions/ln/request.ts

```diff
@@ -43,14 +44,18 @@ const request = async (
       throw new Error("Could not find an allowance for this host");
     }
 
+    if (!accountId) {
```

Comment:
Is this just for TS? When could an `accountId` be falsy?

Reply:
Yes, just for TS

Reply:
We could add a comment in these cases, i.e. `// type guard`